packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.base.tsx

```diff
@@ -90,7 +92,7 @@ export class GroupedListBase extends BaseComponent<IGroupedListProps, IGroupedLi
             items={groups}
             onRenderCell={this._renderGroup}
             getItemCountForPage={this._returnOne}
-            getPageHeight={getGroupHeight && this._getPageHeight(getGroupHeight)}
+            getPageHeight={this._getPageHeight}
```

Comment:
@ThomasMichon would you recommend I use `getPageSpecification` instead? As you mentioned in https://github.com/OfficeDev/office-ui-fabric-react/pull/5204#discussion_r197307727.

Reply:
`getPageSpecification` allows you to return `itemCountForPage` and `pageHeight` in the same call. Though there is a bug right now where if `getPageHeight` is not passed explicitly in `props`, the virtualization screws up.

Reply:
Great, thank you for the context. I will track this in https://github.com/OfficeDev/office-ui-fabric-react/issues/7925.